mycroft/client/wifisetup/main.py

```diff
@@ -38,6 +42,29 @@
 
 LOG = getLogger("WiFiClient")
 
+SCRIPT_DIR = dirname(realpath(__file__))
+
+
+def var_dump(var, prefix=''):
+    my_type = '[' + var.__class__.__name__ + '(' + str(len(var)) + ')]:'
+    print(prefix, my_type)
+    prefix += '    '
+    for i in var:
+        if type(i) in (list, tuple, dict, set):
+            var_dump(i, prefix)
+        else:
+            if isinstance(var, dict):
+                print(prefix, i, ': (', var[i].__class__.__name__, ')', var[i])
+            else:
+                print(prefix, '(', i.__class__.__name__, ')', i)
+
+
+def cli_no_output(*args):
```

Comment:
As for the spoken sentences, the phrases are being specifically designed to walk a novice user through the process.  I may refine them slightly beyond this, but this is a rare-enough event that I have no significant concern that expert users will be irritated by the wordiness.  Before these changes there was no way a user would have been able to perform the wifi setup without reading the source code.